### ISSUE TITLE
Feature Request: On Tap Gesture

### ISSUE THREAD

**Issue by Awesomeplayer165** [TokamakUI/Tokamak#519]
Thank you so much for creating this amazing tool! 
I was wondering if there was any way to check if a View has been touched/tapped by the user? In SwiftUI, I would accomplish this by using a `.onTapGesture {}` on the SwiftUI `View`. Maybe there already is something equivalent and I just haven't found it yet.

Thanks!

**Comment by libewa** [comment]
this would be the equivalent of `onclick=""` in HTML

**Comment by carson-katri** [comment]
Two current solutions are to wrap the View in a `Button` or a `HTML` view with a listener for click events.

All of SwiftUIs gesture APIs still need to be built out.

**Comment by libewa** [comment]
Yeah but `Button`s are ugly.
And with HTML you then can't execute Swift code.

**Comment by carson-katri** [comment]
@libewa Please see this second example in the README of a HTML view calling Swift code on the mouse move event. It could be adapted to onclick fairly easily: https://github.com/TokamakUI/Tokamak#arbitrary-html

**Comment by MrDevel0per** [comment]
I agree we need this feature implimented on its own, without a button.

**Comment by shial4** [comment]
work is being done here 
https://github.com/TokamakUI/Tokamak/issues/530
